index.html

```diff
@@ -39,6 +39,80 @@ <h1>I'm José</br>
             </li>
         </ul>
     </section>
+    <section class="works">
+        <article>
+            <img src="images/Snapshoot Portfolio.svg">
+            <div>
+                <h2>Tonic</h2>
+                <div class="ref">
+                    <p class="category">Canopy</p>
+                    <p class="rest">Back End Dev</p>
+                    <p class="rest">2015</p>
+                </div>
+                <p class="copy">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="r1">
+                    <button class="a1">html</button>
+                    <button class="a1">css</button>
+                    <button class="a1">javascript</button>
+                </div>
```

Comment:
Great job team 🤝

- [ ] Kindly make sure this `<div>` element is actually a `<ul>`, and the children elements of `html`, `css`, and `Javscript` are `<li>` tags instead of `<button>` tags.

Reply:
Kindly make sure this <div> element is actually a <ul>, and the children elements of html, css, and Javscript are <li> tags instead of <button> tags.